script.js

```diff
@@ -0,0 +1,959 @@
+/* eslint-disable no-continue */
+// The above es lint rule is disabled, because if continue is not used
+// most of the for loops will have numerous nested conditions and loops
+// which makes the code less readable.
+
+// Global variables
+// DOM Elements
+// Table container to hold the result of the game.
+let tableScoreCard = null;
```

Comment:
These globals can be organised into groups each separated by a newline for easier readability.